packages/common/src/utils/CommonKnownContentTypes.ts

```diff
@@ -4,7 +4,7 @@
  * @type {Object}
  */
 export const CommonKnownContentTypes = {
-  xre: 'asset/xre',
+  xre: 'prefab/xre',
   gltf: 'model/gltf',
```

Comment:
So, are prefab files a custom file type we are introducing? I thought they were just gltf files?

Reply:
they are the ecs represented in a gltf file